tests/pipeline/test_alignment.py

```diff
@@ -1,127 +0,0 @@
-import pytest
-import random
-
-import numpy as np
-import astropy.wcs
-
-import sqlalchemy as sa
-
-from models.base import SmartSession
-from models.provenance import Provenance
-from models.image import Image
-
-from improc.alignment import ImageAligner
-
-
-def test_warp_decam( decam_datastore, decam_reference ):
```

Comment:
Why are these tests deleted?  Or is this an artifact of the "cleanup" PR not being properly merged?